CHANGELOG.md

```diff
@@ -9,6 +9,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Native function calling support to `OpenAiChatPromptDriver`, `AzureOpenAiChatPromptDriver`, `AnthropicPromptDriver`, `AmazonBedrockPromptDriver`, `GooglePromptDriver`, and `CoherePromptDriver`.
 - `OllamaEmbeddingDriver` for generating embeddings with Ollama.
 - `GriptapeCloudKnowledgeBaseVectorStoreDriver` to query Griptape Cloud Knowledge Bases.
+- `RouteLLMPromptDriver` for using RouteLLM to route between a strong Prompt Driver and weak Prompt Driver. 
```

Comment:
In response to 

> Should we consider other routing solutions before putting this in place?

If you were referencing the choice of making a "composite/decorator" Prompt Driver: I really like the decorator pattern for this scenario. It seems ideal to allow users to use a familiar and existing api (Prompt Driver), especially since LLM routing would primarily be used as a mechanism for reducing costs that would be introduced after the initial business logic. Like first I'd implement my application and get it working, then I'd say wait I want to reduce my LLM costs. At that point all they need to do is swap out the Prompt Driver. Sounds like a good deal!

If you were referencing the choice of `RouteLLM` vs something else, you did put `RouteLLM` in the name, so it seems like nothing is preventing us from adding another Prompt Driver that does LLM routing differently.